tox.ini

```diff
@@ -63,6 +66,14 @@ extras =
     test
 commands =
     pytest -v --durations=5 --maxfail=10 {posargs}
+    pytest -v --durations=5 --maxfail=10 test/synchronous/ {posargs}
```

Comment:
Does running pytest twice generate two XML reports? Which one gets uploaded?

There should be a way to run both of these in one pytest command, no?

Reply:
Good point, I'm not sure how the XML reports interact. I'll open a follow-up PR with a better solution.

Reply:
Have you opened a ticket for this yet? The second XML file overwrites the first and there are actually other problems too. Like any test that relies on `posargs` gets run twice, for example the test_auth_oidc.py file gets run twice: https://evergreen.mongodb.com/task/mongo_python_driver_oidc_auth_test__platform~macos_oidc_auth_test_patch_636603f89371e626c41e48504c986e53f5122aaf_6696cbe31505d30007b20aae_24_07_16_19_39_11

Reply:
I haven't, sorry--this slipped past me. I don't know if there is a way to run this all in one command while keeping the existing behavior: we want to run all tests in `test/` and `test/asynchronous` by default, but override both those directories with whatever tests are specified.

Reply:
No problem. Opened https://jira.mongodb.org/browse/PYTHON-4558.